libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -105,22 +105,23 @@ export const OcmClusterDetailsFormFields = ({
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
   const cpuArchitecture = (architectureData[values.cpuArchitecture] as CpuArchitectureItem).label;
-  const externalPartnerIntegrationsCheckboxState = useExternalPartnerIntegrationsCheckboxState(
+  const oracleDropdownItemState = useOracleDropdownItemState(
     clusterExists,
     featureSupportLevelData,
   );
+  const [customManifestsCheckboxDisabled, setCustomManifestsCheckboxDisabled] =
+    React.useState<boolean>(false);
 
   React.useEffect(() => {
     nameInputRef.current?.focus();
   }, []);
 
-  const handleExternalPartnerIntegrationsChange = React.useCallback<
-    NonNullable<OcmCheckboxFieldProps['onChange']>
-  >(
-    (value: boolean, _event: React.FormEvent<HTMLInputElement>) => {
-      const checked = Boolean(value);
-      setFieldValue('addCustomManifest', checked, false);
-      clusterWizardContext.setAddCustomManifests(checked);
+  const handleExternalPartnerIntegrationsChange = React.useCallback(
+    (selectedPlatform: ExternalPlatformType) => {
+      const isOracleSelected = selectedPlatform === 'oci';
+      setFieldValue('addCustomManifest', isOracleSelected, false);
```

Comment:
If users want to use custom manifests, we should not be unchecking that for them.

We should only make it enabled when users select OCI, and leave it unchanged otherwise.